docs/en/source/onepageTutorial.rst

```diff
@@ -0,0 +1,327 @@
+*****************************************
+Beginners Guide
```

Comment:
Right now, this seems to be a copy of the introduction document. I think as a one page tutorial it is too detailed and, of course, the other comments made to the above also apply here.